### DISCUSSION TITLE
RC11 throws missing ref

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#5820] [Q&A]
Any Idea? RC8 works fine but after getting the error below

2024/11/07 23:19:05 Failed to apply migration 1717233558_v0.23_migrate3.go: recordRef: Missing or invalid collection..

**Comment by ganigeorgiev** [comment]
Hm, this is most likely because of the rc10 system collections id change that I seems to have forgotten to reload the collections definition cache. I'll push a fix sometime later today.

**Reply by ganigeorgiev** [reply]
The validation error should be fixed in [v0.23.0-rc12](https://github.com/pocketbase/pocketbase/releases/tag/v0.23.0-rc12).
